src/init.cpp

```diff
@@ -480,7 +480,7 @@ void SetupServerArgs()
                    ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
     argsman.AddArg("-externalip=<ip>", "Specify your own public address",
                    ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
-    argsman.AddArg("-onlynet=<net>", "Only connect to nodes in network <net> (IPv4, IPv6 or Tor)",
+    argsman.AddArg("-onlynet=<net>", "Make outgoing connections only through network <net> (IPv4, IPv6 or Tor). Incoming connections are not affected by this option. This option can be specified multiple times to allow multiple networks.",
                    ArgsManager::ALLOW_ANY, OptionsCategory::CONNECTION);
```

Comment:
Can you break the long string to the next line to avoid a really long line here?

Where the period is after Tor). You can do Tor. " then start the next line with "Incoming connections...", another return, then ArgsManager::ALLOW_ANY...

Reply:
is this good?